docs/using-hive.md

```diff
@@ -77,87 +234,77 @@ metadata:
+      name: mycluster-ssh-key
+  pullSecretRef:
     name: mycluster-pull-secret
-  sshKey:
-    name: mycluster-ssh-key
 ```
 
-The hiveutil CLI (see `make hiveutil`) offers a create-cluster command for generating a cluster deployment and submitting it to the Hive cluster using your current kubeconfig.
-
-To view what create-cluster generates, *without* submitting it to the API server, add `-o yaml` to the above command. If you need to make any changes not supported by create-cluster options, the output can be saved, edited, and then submitted with `oc apply`.
+For Azure, replace `spec.platform` with:
 
-By default this command assumes the latest Hive master CI build, and the latest OpenShift stable release. `--release-image` can be specified to control which OpenShift release image to install in the cluster.
-
-#### Create Cluster on AWS
-
-Credentials will be read from your AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables. Alternatively you can specify an AWS credentials file with --creds-file.
-
-```bash
-bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=aws mycluster
+```yaml
+azure:
+  baseDomainResourceGroupName: my-bdrgn
+  credentialsSecretRef:
+    name: mycluster-azure-creds
+  region: centralus
 ```
 
-#### Create Cluster on Azure
+For GCP, replace `spec.platform` with:
 
-Credentials will be read from `~/.azure/osServicePrincipal.json` typically created via the `az login` command.
-
-```bash
-bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=azure --azure-base-domain-resource-group-name=myresourcegroup mycluster
+```yaml
+gcp:
+  credentialsSecretRef:
+    name: mycluster-gcp-creds
+  region: us-east1
 ```
 
-#### Create Cluster on GCP
+### Machine Pools
 
-Credentials will be read from `~/.gcp/osServiceAccount.json`, this can be created by:
+To manage `MachinePools` Day 2, you need to define these as well. The definition of the worker pool should mostly match what was specified in `InstallConfig` to prevent replacement of all worker nodes.
 
- 1. Login to GCP console at https://console.cloud.google.com/
- 1. Create a service account with the owner role.
- 1. Create a key for the service account.
- 1. Select JSON for the key type.
- 1. Download resulting JSON file and save to `~/.gcp/osServiceAccount.json`.
+`InstallConfig` is limited to the one worker pool, but Hive can sync additional machine pools Day 2.
 
-```bash
-bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=gcp mycluster
+```yaml
+apiVersion: hive.openshift.io/v1
+kind: MachinePool
+metadata:
+  creationTimestamp: null
+  name: baremetal-demo-worker
+spec:
+  clusterDeploymentRef:
+    name: baremetal-demo
+  name: worker
+  platform:
+    azure:
+      osDisk:
+        diskSizeGB: 128
+      type: Standard_D2s_v3
+    gcp:
+      type: n1-standard-4
+    aws:
+      rootVolume:
+        iops: 100
+        size: 22
+        type: gp2
+      type: m4.xlarge
+  replicas: 0
```

Comment:
Use the same replicas count that was used in the ClusterDeployment example.

Reply:
done